.github/workflows/jane_syntax.yml

```diff
@@ -0,0 +1,42 @@
+name: jane-syntax-upstream-build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        ocaml-compiler:
+          - "4.14.0"
+
+    steps:
+    - name: Checkout the Flambda backend repo
+      uses: actions/checkout@master
+      with:
+        path: 'flambda_backend'
+
+    - name: Setup OCaml ${{ matrix.ocaml-compiler }}
+      uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: ${{ matrix.ocaml-compiler }}
+
+    - name: Try building Jane_syntax and its dependencies with upstream OCaml
+      working-directory: flambda_backend
+      run: |
+        # Stub [Language_extension] in the same way we do internally.
+        echo "include Language_extension_kernel let is_enabled _ = true let is_at_least _ _ = true" > language_extension.ml
+        if ! \
+          opam exec -- ocamlc -stop-after typing \
+            -I +compiler-libs -I ocaml/utils -I ocaml/parsing \
+            ocaml/utils/language_extension_kernel.{mli,ml} \
+            language_extension.ml \
+            ocaml/parsing/jane_asttypes.mli \
+            ocaml/parsing/{jane_syntax_parsing,jane_syntax}.{mli,ml}
```

Comment:
```suggestion
            -I +compiler-libs -I utils -I parsing \
            utils/language_extension_kernel.{mli,ml} \
            language_extension.ml \
            parsing/jane_asttypes.mli \
            parsing/{jane_syntax_parsing,jane_syntax}.{mli,ml}
```

Reply:
Resolved by factoring out the command into a script.